src/scss/components/productItem.scss

```
@@ -0,0 +1,79 @@
+.product {
+	&__item {
+		border: 1px $border-color solid;
+		margin-bottom: 3rem;
+		&:hover {
+			.product {
+				&__widget {
+					top: 0%;
+				}
+			}
+		}
+	}
+
+	&__image {
+		position: relative;
+		overflow: hidden;
+	}
+
+	&__discount {
+		@include absolute(2rem, "", "", 3rem);
+		z-index: 1;
+		background-color: $light-red;
+		@include centerWidthHeight(4.5rem);
+		@include border-radius(50%);
```

```suggestion
		@include absolute(2rem, "", "", 3rem);
		@include centerWidthHeight(4.5rem);
		@include border-radius(50%);
		z-index: 1;
		background-color: $light-red;
```
Chị đưa hết include lên trên đầu cho dễ theo dõi nhé

---

okie em